src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -139,6 +190,31 @@ public void setGoalSpeeds(ChassisSpeeds goalSpeeds, boolean fieldCentric) {
         this.setControl(request);
     }
 
+    // SYS ID
+
+    public void setSysIdRoutine(SysIdRoutineType routineType) {
```

Comment:
I might just be missing it, but where is this actually called to allow the operator to choose the routine?

Reply:
it hasnt been called at all yet, I was going to talk to you about how you think it should be called (like button binding or constants etc)

Reply:
Since we are adding it here, we might as well call it - This seems like something that should be configured with buttons if possible (since it would only be available in test mode). Also we should send logs of what routine we are running to advantage scope